mobilizon_reshare/config/config.py

```diff
@@ -130,3 +130,7 @@ def clear(cls):
 
 def get_settings():
     return CustomConfig.get_instance().settings
+
+
+def get_settings_without_validation():
+    return build_settings()
```

Comment:
Is this really necessary? Maybe we should consider renaming the original function?

Reply:
this is necessary and I think we want to use it more in testing. It saves us from providing full config files all the time when we know we care only about a key or two